pkg/helpers/homebrew.go

```diff
@@ -0,0 +1,60 @@
+package helpers
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/pior/dad/pkg/env"
+	"github.com/pior/dad/pkg/utils"
+)
+
+type Homebrew struct {
+	prefix string
+}
+
+// NewHomebrew is returning a new Cellar
+func NewHomebrew() *Homebrew {
+	return NewHomebrewWithPrefix("/usr/local")
+}
+
+func NewHomebrewWithPrefix(prefix string) *Homebrew {
+	if !utils.PathExists(filepath.Join(prefix, "Cellar")) {
+
+		path := strings.Split(env.NewFromOS().Get("PATH"), ":")
+
+		if len(path) > 0 && utils.PathExists(path[0]) {
+			prefix = filepath.Dir(path[0])
+		}
+	}
+
+	return &Homebrew{prefix: prefix}
+}
+
+func pathToPackage(filename string) string {
```

Comment:
Just a thought: maybe the helpers should get their own package (`pkg/helpers/homebrew`) to scope function like this `pathToPackage()`.
Otherwise we should either namespace it with `homebrew` or attach it to `Homebrew`